cypress/support/api_commands.js

```diff
@@ -177,6 +177,44 @@ Cypress.Commands.add('apiAddUserToTeam', (teamId, userId) => {
     });
 });
 
+/**
+ * Join teammates directly via API
+ * This API assume that the user is logged in and has cookie to access
+ * @param {String} name - The name arg previously used during team-create (unique but not GUID)
+ * @param {Integer} size - The amount of users to join
+ * All parameter required
+ */
+Cypress.Commands.add('apiBatchJoinTeam', (name, size = 60) => {
+    return cy.getCookie('MMUSERID').then((cookie) => {
+        const admin = cookie.value;
+        cy.request({method: 'GET', url: `/api/v4/teams/name/${name}`}).then((teamResp) => {
+            const key = teamResp.body.id;
+            cy.request({method: 'GET', url: `/api/v4/users?per_page=${size}`}).then((userResp) => {
+                // Shape the users list into batch request body, skip the team admin.
+                const ulist = userResp.body.filter((u) => u.id !== admin).map((user) => {
```

Comment:
Instead of fetching the users here, I think it'd be better to feed the list in as an argument and add those specific users to the team. `ulist` can also just be `usersList`.

Reply:
The 60 value was due to the API batch call stopping at 59 for some reason. I don't have a very good explanation for it. It really is supposed to be +1 which can be verified for team sizes lower than 59. Regarding the list, yes I made the change to accept the list as a argument in today's PR.